### PR TITLE
Feature/ht 36/create booking form fields

### PR DESCRIPTION
**Pull request by kalibria**
user can create new ride filling out start - end date and time of the ride.

### CODE REVIEW

File: src/auth/components/loginForm/LoginForm.tsx [hunk 1]
```diff
@@ -34,7 +35,13 @@ const LoginForm = () => {
 
   useEffect(() => {
     if (resultUser.isSuccess) {
-      dispatch(setCarId(resultUser.data.user.availableCars));
+      batch(() => {
+        batch(() => {
```
Comment (ikushlianski) [inline src/auth/components/loginForm/LoginForm.tsx]: do we need double batch function?

File: src/createNewBooking/components/CreatingNewBooking.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      .typeError('Заполните поле')
+      .required('Заполните поле'),
+    endTime: Yup.date()
+      .typeError('Заполните поле')
+      .required('Заполните поле')
+      .when('startTime', (startTime: Dayjs) => {
+        return Yup.date()
+          .min(
+            startTime,
+            'Время завершения поездки должна быть больше времени начала поездки',
```
Comment (ikushlianski) [inline src/createNewBooking/components/CreatingNewBooking.tsx]: должно

Comment (kalibria) [reply src/createNewBooking/components/CreatingNewBooking.tsx]: corrected

File: src/auth/components/loginForm/LoginForm.tsx (hunk 1 above)
Comment (kalibria) [reply src/auth/components/loginForm/LoginForm.tsx]: removed superfluous

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**